HoloRepositoryDemoApplication/Assets/Scripts/HologramList.cs

```diff
@@ -9,27 +9,32 @@ public class HologramList : MonoBehaviour
     [SerializeField]
     private GameObject buttonTemplates = null;
     [SerializeField]
+    private TextMeshProUGUI Title = null;
+    [SerializeField]
     private TextMeshProUGUI Message = null;
+
     public static bool InitialFlag = true;
     public static bool SceneSwitchFlag = false;
     public static List<Hologram> hologramList = new List<Hologram>();
-    public static string patientID = string.Empty;
+    public static Patient Patient;
 
     public void Start()
     {
         if (InitialFlag)
         {
+            Title.text = "Hologram List";
             Message.text = "Please select a patient to check the Hologram";
```

Comment:
We don't need to capitalise hologram. Maybe "Please select a patient to browse available holograms"